### PR TITLE
Add org.opentransactions.metier

### PR DESCRIPTION
**Pull request by justusranvier**
### Please confirm your submission meets all the criteria

- [Y] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [Y] My pull request follows the instructions at [App Submission][submission].
- [Y] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [Y] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [Y] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [Y] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [N/A] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by justusranvier** [comment]
bot, build org.opentransactions.metier

**Comment by justusranvier** [comment]
bot, build org.opentransactions.metier

File: org.opentransactions.metier.json [hunk 1]
```diff
@@ -0,0 +1,215 @@
+{
+  "id": "org.opentransactions.metier",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "runtime-commit": "3522f785effad84e56b127dc800e2982ae4eeca621d7b12a1a26136039d452ae",
```
Comment (barthalion) [inline org.opentransactions.metier.json:5]:
```suggestion
```

File: org.opentransactions.metier.json [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+  "sdk": "org.kde.Sdk",
+  "sdk-commit": "fac62c60e263c7604c26ea9cf86530e59837bdb5c09b165d0451720644d24e2e",
```
Comment (barthalion) [inline org.opentransactions.metier.json:7]:
```suggestion
```

File: org.opentransactions.metier.json [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+          "type": "git",
+          "disable-shallow-clone": true
+        }
+      ]
+    }
+  ],
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
```
Comment (barthalion) [inline org.opentransactions.metier.json:211]:
```suggestion
```

File: org.opentransactions.metier.json [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+          "url": "https://github.com/Open-Transactions/metier",
+          "tag": "21.1",
+          "commit": "1932dd703563f4646cb30fcd3f286745575226a9",
+          "type": "git",
+          "disable-shallow-clone": true
+        }
+      ]
+    }
+  ],
+  "finish-args": [
```
Comment (barthalion) [inline org.opentransactions.metier.json:208]: We traditionally put it before `modules` but it's a nitpick.

**Comment by barthalion** [comment]
```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri)
libGL error: failed to load driver: amdgpu
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: radeonsi
```

You may want to add `--device=dri`.

File: org.opentransactions.metier.json (hunk 1 above)
Comment (justusranvier) [reply org.opentransactions.metier.json:5]: It's strange that I don't remember adding that line when I wrote the manifest. Did flatpak-builder do that?

**Comment by justusranvier** [comment]
bot, build org.opentransactions.metier

**Pull request closed without merging**